helm/dagster/values.yaml

```diff
@@ -34,6 +34,8 @@ dagit:
 
   service:
     type: ClusterIP
+    # Defines the port where Dagit will serve requests; if changed, don't forget to update the
```

Comment:
Ideally, this port setting would automatically cascade the probe port values so they're always in sync, but I didn't see a clear path to make that happen, so I added this "doc" note.